packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.html

```diff
@@ -1,3 +1,8 @@
 <template>
-  <a href={result.clickUri} target={target}>{displayedTitle}</a>
-</template>
\ No newline at end of file
+  <template if:true={isSalesforceLink}>
+    <a href="#" onclick={handleOpenTabSalesforceLink}>{displayedTitle}</a>
```

Comment:
from an accessibility standpoint it might be better to add an aria-label that describes that it'll be opened in a salesforce tab. I think an anchor tag with no href isn't considered accessible.